hazelcast/src/main/java/com/hazelcast/spi/properties/GroupProperty.java

```diff
@@ -536,6 +536,9 @@
      * Back pressure is implemented by making asynchronous backups operations synchronous. This prevents the internal queues from
      * overflowing because the invoker will wait for the primary and for the backups to complete. The frequency of this is
      * determined by the sync-window.
+     *
+     * To deal with overloads of backups, the property 'hazelcast.operation.backup.timeout.millis' should be set to a larger
+     * value; above 60000 is recommended. Otherwise it can still happen backups accumulate.
```

Comment:
```
     * To deal with overloads of backups the property {@link #OPERATION_BACKUP_TIMEOUT_MILLIS} should be set to a larger value;
     * above {@code 60000} is recommended. Otherwise it can still happen that backups accumulate.
```